src/style.css

```diff
@@ -0,0 +1,125 @@
+@font-face {
+  font-family: Avenir-book;
+  src: url(./fonts/Avenir-Book.ttf) format("truetype");
+  font-weight: lighter;
+}
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf) format("truetype");
+  font-weight: bold;
+}
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc) format("truetype");
+  font-weight: normal;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  padding: 0;
+}
+
+.conteiner-1 {
+  display: flex;
+  align-items: center;
+  padding: 0;
+  margin-top: 20px;
+  height: 70px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.conteiner-1:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.conteiner-2 {
+  display: flex;
+  align-items: center;
+  padding: 0;
+  margin-top: 20px;
+  height: 42px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.conteiner-2:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.bar-1 {
+  width: 100%;
+  border: none;
+  outline: none;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.bar-1::placeholder {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+.bar-1:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-family: "Avenir-Heavy", sans-serif;
+  line-height: 22px;
+}
+
+.bar-2 {
+  width: 100%;
+  padding: 0;
+  border: none;
+  outline: none;
+  font-family: "Avenir", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 19px;
+}
```

Comment:
you have some similar styles in `.bar-1` selector. you can create a common class `.input` and merge the same styles.

Reply:
same for other elements (`form` and `img`)